nucliadb_client/nucliadb_client/knowledgebox.py

```diff
@@ -133,16 +143,47 @@ def delete(self):
         resp = self.http_manager_v1.delete("")
         return resp.status_code == 200
 
+    async def import_tar_bz2(self, filename):
+        with tarfile.open(filename, mode="r:bz2") as tar:
+            for member in tar.getmembers():
+                buffer = tar.extractfile(member.name)
+
+                async def upload_generator(buffer: BytesIO, member: tarfile.TarInfo):
+                    chunk_size = 1_000_000
+                    buffer.seek(0)
+                    count = 0
+                    ubd = UploadBinaryData()
+                    ubd.count = count
+                    ubd.metadata.size = member.size
+                    ubd.metadata.kbid = self.kbid
+
+                    # Tweak the key so that it points to the new kbid
+                    exported_kbid = member.name.split("/")[1]
+                    ubd.metadata.key = member.name.replace(exported_kbid, self.kbid)
```

Comment:
@bloodbare do you think this is OK to add here on the client side while importing?